### DISCUSSION TITLE
希望出一个可以自动推算到某个时间点之前大概能攒多少抽的功能

### DISCUSSION THREAD

**Discussion by Lifeqian** [MaaAssistantArknights/MaaAssistantArknights#11393] [Ideas]
想知道自己新年限定前能不能攒够300井，看攒抽规划又看不太懂。心想要是maa可以根据已有的仓库推算大致能攒多少抽，然后加上一些赠送的抽，看能不能攒够井就好了。不然每次限定还没来就去抽标准还没有的干员了。

**Comment by Windsland52** [comment]
https://ark.yituliu.cn/tools/gachaCalc
